### DISCUSSION TITLE
How to add disappear class for animate?

### DISCUSSION THREAD

**Discussion by victorpavlenko** [ReactTooltip/react-tooltip#1103] [Q&A] [answered]
I can work with .react-tooltip__show for appear animation, any ideas how to add disappear class?

**Comment by gabrieljablonski** [comment]
This will depend on what you're trying to accomplish.

If you have a look at [the docs about the tooltip styling](https://react-tooltip.com/docs/examples/styling#core-styles), you'll see more details about how it works internally.

The default fade-in animation is done with `transition: opacity 0.3s ease-out;`

**Reply by victorpavlenko** [reply]
I don't think this works. If you try this from your own link, you will see that the disappearing animation does not work.

I would like to add a disappearing animation, as described above. Perhaps some kind of `_hide` class

https://github.com/ReactTooltip/react-tooltip/assets/5157416/32683df5-064b-4468-8950-39135cb775f6


Foe example

**Reply by gabrieljablonski** [reply, marked as answer]
Due to a slight design limitation on the tooltip, it is not easy to do a "closing animation", since the tooltip closes instantly when you hover out of it. I've opened the feature request #1105 to track this.
